compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -29,21 +30,22 @@ import Development.IDE.Core.API.Testing
 import Development.IDE.Core.Service.Daml(VirtualResource(..))
 
 main :: IO ()
-main = SS.withScenarioService LF.versionDefault Logger.makeNopHandle scenarioConfig $ \scenarioService -> do
-  -- The scenario service is a shared resource so running tests in parallel doesn’t work properly.
+main = SS.withScenarioService LF.versionDefault Logger.makeNopHandle scenarioConfig $ \scenarioService ->
+       SS.withScenarioService LF.versionDev     Logger.makeNopHandle scenarioConfig $ \scenarioServiceDev -> do
```

Comment:
I needed to do this since interfaces aren't supported before `1.dev`. If you can think of a better way to test this without spinning up a separate scenario service please let me know!

Reply:
Seems ok for now. Maybe worth adding a TODO and linking to the issue for LF 1.15, so we can revert to using just one scenario service once that becomes the default.

Reply:
will do, thank you!

Reply:
That’s the best option atm. In theory, we could setup the scenario service to run multiple LF versions in separate contexts but atm this is not supported.